src/app/cli/src/coda_worker_testnet.ml

```diff
@@ -24,7 +26,8 @@ module Api = struct
           (** The int counts the number of ongoing RPCs. when it is 0, it is safe to take the worker offline.
         [stop] below will set the status to `Off, and we only try doing an RPC if the status is `On,
         so eventually the counter _must_ become 0, ensuring progress. *)
-    }
+    ; root_lengths: int Array.t
```

Comment:
We now have 6 array fields, that represents a certain state of worker. It would be good to close these fields into one field and not have to get them by the index of a worker.